src/winetricks

```diff
@@ -12114,6 +12114,7 @@ w_metadata vcrun2017 dlls \
     publisher="Microsoft" \
     year="2017" \
     media="download" \
+    conflicts="vcrun2019" \
```

Comment:
Nice catch, but shouldn't it also include vcrun2015?

Reply:
I don't think newer versions conflict with older ones. The vcrun installers don't seem to complain about already installed vcrun2015 and I am pretty sure I've seen both vcrun2015 and vcrun2017 installed on actual Windows machines.

Reply:
They both install mfc140.dll..

Reply:
Yeah, which gets replaced with a newer version when you install a newer visual c++ runtime.

Reply:
On windows; winetricks will detect that the file is already there and bail out unless --force is used.

Reply:
I mean, I don't mind adding it in the conflict, but it could be useful to have an upgrade path for existing prefixes for whatever potential reason users might have.

Reply:
Oh, I see, didn't know that :D Will add the conflict then.

Reply:
It's out of the scope of this PR, but it may be doable to do like we do in newer .Net versions and touch a file (e.g., c:/windows/vcrun2019.installed) and use that for the installed file rather than mfc140.dll. That assumes of course that the installer's overwrite feature works..